src/app/Shared/Model/Client_Isolate.model.ts

```diff
@@ -1,13 +1,53 @@
-export type DbCollection = {
-    microorganism: string;
-    samplingContext: string;
-    matrix: string;
-    federalState: string;
-    samplingStage: string;
-    origin: string;
-    category: string;
-    productionType: string;
-}[];
+export type ResistantValues =
+    | "amp"
+    | "azi"
+    | "chl"
+    | "cip"
+    | "cli"
+    | "col"
+    | "dap"
+    | "ery"
+    | "etp"
+    | "fep"
+    | "fop"
+    | "fox"
+    | "fus"
+    | "f_c"
+    | "gen"
+    | "imi"
+    | "kan"
+    | "lzd"
+    | "mero"
+    | "mup"
+    | "nal"
+    | "pen"
+    | "rif"
+    | "smx"
+    | "str"
+    | "syn"
+    | "taz"
+    | "tec"
+    | "tet"
+    | "tgc"
+    | "tia"
+    | "tmp"
+    | "trm"
+    | "t_c"
+    | "van";
+
+export type DbStringKey =

```

Comment:
whats a DbStringKey? Perhaps you mean something like DbPrimaryAttributeKey or something?
Can be simlified: `= Exclude<DbKey, "resistance">`

Reply:
Done. Commit "Simplify code"